utils/relaxng.xml

```diff
@@ -32,9 +32,7 @@
     <oneOrMore>
       <element name="target">
         <attribute name="host">
-          <data type="string">
-            <param name="pattern">(([A-Za-z0-9äö-]+|\*)\.)*([A-Za-z0-9äö-]+|\*)</param>
-          </data>
+          <data type="string" />
```

Comment:
I haven't had a chance to look into HTTPS Everywhere's overall handling of IDNs, but removing the regex check here is definitely too much. If the goal is to include all alphanumeric characters in Unicode, I think you want to use one of the Posix character classes, problably `:alnum:`: http://www.regular-expressions.info/posixbrackets.html. Don't forget to also include `-` which is a valid character in DNS names.